### PR TITLE
feat: add blog & service & testimonials models

### PR DESCRIPTION
**Pull request by CoderMungan**

### CODE REVIEW

File: company/apps/blog/models.py [hunk 1]
```diff
@@ -1,3 +1,47 @@
 from django.db import models
+from apps.main.mixin import TimeBasedStampModel
+from autoslug import AutoSlugField
+from tinymce.models import HTMLField
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Category(TimeBasedStampModel):
+  name = models.CharField(("Category Name"), max_length=50)
+  slug = AutoSlugField(
```
Comment (enescomruk72) [inline company/apps/blog/models.py:9]: Here populate_from should be name, not title

File: company/apps/blog/models.py [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+        populate_from="title", editable=False, always_update=True, blank=True
+    )
+
+  class Meta:
+    verbose_name = "Blog"
+    verbose_name_plural = "Blogs"
+
+  def __str__(self) -> str:
+    return self.title
```
Comment (enescomruk72) [inline company/apps/blog/models.py:47]: gj

File: company/apps/service/models.py [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+  short_content = models.CharField(("Short Content"), max_length=127)
+  content = HTMLField(("Content"))
+  icon = models.CharField(("Icon"),choices=ICON_CHOICES, max_length=150)
+
+  class Meta:
+    verbose_name = "Service"
+    verbose_name_plural = "Services"
+
+  def __str__(self) -> str:
+      return self.title
```
Comment (enescomruk72) [inline company/apps/service/models.py:27]: ok

File: company/apps/testimonials/models.py [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+  image = models.ImageField(("Image"), upload_to="testimoninials/", height_field=None, width_field=None, max_length=None)
+  star = models.PositiveIntegerField(("Star Point"),default = 1, validator = [MaxValueValidator(5)])
+  content = HTMLField(("Testimonial Content"), max_length = 400)
+
+  class Meta:
+    verbose_name = "Testimonial"
+    verbose_name_plural = "Testimonials"
+
+  def __str__(self) -> str:
+      return self.author
```
Comment (enescomruk72) [inline company/apps/testimonials/models.py:18]: ok for now

**Review by enescomruk72 — CHANGES_REQUESTED** [review]
slug = AutoSlugField(
        populate_from="title", editable=False, always_update=True, blank=True
    )
    
    Here populate_from should be name, not title

File: company/apps/blog/models.py (hunk 1 above)
Comment (CoderMungan) [reply company/apps/blog/models.py:9]: [fb0afc6](https://github.com/RythmTechnologies/Company-Promotion/pull/6/commits/fb0afc6bcbd20ac86efb30895dd1ffe12b4a8acb) ow i see that and changed !

File: company/apps/blog/models.py (hunk 2 above)
Comment (CoderMungan) [reply company/apps/blog/models.py:47]: ty

File: company/apps/testimonials/models.py (hunk 4 above)
Comment (CoderMungan) [reply company/apps/testimonials/models.py:18]: ty

**Comment by CoderMungan** [comment]
> slug = AutoSlugField( populate_from="title", editable=False, always_update=True, blank=True )
> 
> ```
> Here populate_from should be name, not title
> ```

[fb0afc6](https://github.com/RythmTechnologies/Company-Promotion/pull/6/commits/fb0afc6bcbd20ac86efb30895dd1ffe12b4a8acb)

File: company/apps/blog/models.py [hunk 5]
```diff
@@ -1,3 +1,47 @@
...
```
Comment (mciray) [inline company/apps/blog/models.py:47]: nice job

File: company/apps/service/models.py (hunk 3 above)
Comment (mciray) [inline company/apps/service/models.py:27]: well done

**Review by mciray — APPROVED** [review]
lgtm

**Pull request merged**